.github/workflows/sarplus.yml

```
@@ -36,10 +36,10 @@ jobs:
     # Test pysarplus with different versions of Python.
     # Package pysarplus and upload as GitHub workflow artifact when merged into
     # the main branch.
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10"]
```

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```
Do you think that it will also work on 3.11?

---

I remember I tried 3.11 before, but there are some dependencies problems, so I removed python 3.11 and focused on fixing the problems on other python versions.  I'll try python 3.11 later.